src/app/home/problem-list/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,90 @@
+<div class="containerX">
+  <div class="container">
+    <div fxLayout="row" fxLayoutAlign="space-around center">
+      <h1>Welcome {{ userInfo.displayName }} !</h1>
```

Comment:
Could you also please remove the space after the name? `<h1>Welcome {{ userInfo.displayName }}!</h1>` looks better in my opnion.